### PR TITLE
ISSUE-816: Spanish translation for menu entry Fluent->overview.

### PR DESCRIPTION
**Pull request by javieretxarri**
Also creating two files that are being referenced from overview file

### CODE REVIEW

File: docs/fluent/overview.es.md [hunk 1]
```diff
@@ -0,0 +1,591 @@
...
+
+Una vez que se define la migración, se debe de informar a Fluent sobre ella agregándola a `app.migrations` en `configure.swift`.
+
+```swift
+app.migrations.add(CreateGalaxy())
+```
+
+### Migrar
+
+Para ejecutar migraciones, ejecutar `swift run App migrate` desde la línea de comandos o agregar `migrate` como argumento al esquema de la aplicación en Xcode.
```
Comment (jacostaf10) [inline docs/fluent/overview.es.md:316]:
```suggestion
Para realizar migraciones, ejecuta `swift run App migrate` desde la línea de comandos o agrega `migrate` como argumento al esquema de la aplicación en Xcode.
```

File: docs/fluent/overview.es.md [hunk 2]
```diff
@@ -0,0 +1,591 @@
...
+Migration successful
+```
+
+## Consultando
+
+Ahora que el modelo ha sido creado exitosamente y migrado a la base de datos, es el momento de hacer la primera consulta.
+
+### All
+
+Observar la siguiente ruta que devolverá una matriz de todas las galaxias en la base de datos.
```
Comment (jacostaf10) [inline docs/fluent/overview.es.md:334]:
```suggestion
La siguiente ruta devolverá una colección de todas las galaxias en la base de datos.
```

File: docs/fluent/overview.es.md [hunk 3]
```diff
@@ -0,0 +1,591 @@
...
+
+```swift
+final class Galaxy: Model, Content {
+    ...
+}
+```
+
+`Galaxy.query` se utiliza para crear un nuevo generador de consultas para el modelo.  `req.db` es una referencia a la base de datos predeterminada de la aplicación. Por último, `all()` devuelve todos los modelos almacenados en la base de datos.
+
+Si tras compilar y ejecutar el proyecto, se realiza una solicitud de tipo `GET /galaxies`, se devolverá una matriz vacía. Por lo tnato, se tendrá que agregar una ruta para crear una nueva galaxia.
```
Comment (jacostaf10) [inline docs/fluent/overview.es.md:352]:
```suggestion
Si tras compilar y ejecutar el proyecto, se realiza una solicitud de tipo `GET /galaxies`, se devolverá una colección vacía. Por lo tanto, se tendrá que agregar una ruta para crear una nueva galaxia.
```

File: docs/fluent/overview.es.md [hunk 4]
```diff
@@ -0,0 +1,591 @@
...
+
+```swift
+app.post("galaxies") { req async throws -> Galaxy in
+    let galaxy = try req.content.decode(Galaxy.self)
+    try await galaxy.create(on: req.db)
+    return galaxy
+}
+```
+
+En este caso, la versión async no devuelve nada, pero devolverá una vez que se complete el guardado.
```
Comment (jacostaf10) [inline docs/fluent/overview.es.md:381]:
```suggestion
En este caso, la operación async no tiene valor de retorno. La instancia se devolverá una vez se complete el guardado.
```

File: docs/fluent/overview.es.md [hunk 5]
```diff
@@ -0,0 +1,591 @@
...
+Devolverá el modelo creado con un identificador como respuesta.
+
+```json
+{
+    "id": ...,
+    "name": "Milky Way"
+}
+```
+
+Ahora, si se consulta otra vez `GET /galaxies`, se debería de ver la galaxia recién creada devuelta en la matriz.
```
Comment (jacostaf10) [inline docs/fluent/overview.es.md:404]:
```suggestion
Ahora, si se consulta otra vez `GET /galaxies`, se debería de ver la galaxia recién creada devuelta en la colección.
```

File: docs/fluent/overview.es.md [hunk 6]
```diff
@@ -0,0 +1,591 @@
...
+
+{
+    "name": "Sun",
+    "galaxy": {
+        "id": ...
+    }
+}
+```
+
+Deberíade devolverse la creación de la nueva estrella con un identificador único.
```
Comment (jacostaf10) [inline docs/fluent/overview.es.md:523]:
```suggestion
Debería devolver la creación de la nueva estrella con un identificador único.
```

Comment (javieretxarri) [reply docs/fluent/overview.es.md]: @jacostaf10 all the changes done and pushed.
Thanks

**Comment by javieretxarri** [comment]
@jacostaf10 All the changes have been done. Thanks

**Comment by alemohamad** [comment]
@javieretxarri Do you agree about the three suggestions made by @jacostaf10 ? For me they are ok not to do them, but let us know. 😉

**Comment by javieretxarri** [comment]
@alemohamad I will do it today or tomorrow morning. Sorry,  because I didn't realice that @jacostaf10 added more changes

**Review by alemohamad — APPROVED** [review]

**Pull request merged**